Backpack/src/main/java/net/skyscanner/backpack/button/BpkButton.kt

```diff
@@ -176,6 +164,7 @@ open class BpkButton(
     this.iconPosition = iconPosition
     this.minHeight = resources.getDimensionPixelSize(size.minHeight)
     this.iconSize = resources.getDimensionPixelSize(size.iconSize)
+    this.progress.centerRadius = iconSize / 2f - this.progress.strokeWidth
```

Comment:
Progress size is calculated differently – to match it with our icon (and Compose impl version size) we need to substract the stroke width.